src/messenger_bot.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask, request
+import requests
+from pymessenger import Bot
+
+import tokens
+from bot_functions import process_message
+
+app = Flask(__name__)
+
+# FB_API_URL = "https://graph.facebook.com/v2.6/me/messages"
+PAGE_ACCESS_TOKEN = tokens.messenger_bot_page_access_token
+VERIFY_TOKEN = tokens.user_verify_token
+
+bot = Bot(PAGE_ACCESS_TOKEN)
+
+
+@app.route("/", methods=["POST", "GET"])
+def webhook():
+
+    if request.method == "GET":
+        """Veryfy user tokens equality"""
+        if request.args.get("hub.verify_token") == VERIFY_TOKEN:
+            return request.args.get("hub.challenge")
```

Comment:
Do not hardcode values, move them to constants - applies to all hardcoded values - below as well